schemas/blocks/list_collector_content.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$id": "/blocks/list_collector_content.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "summary": {
+    "type": "object",
+    "properties": {
+      "item_title": {
+        "$ref": "../string_interpolation/definitions.json#/string_with_placeholders"
+      },
+      "title": {
+        "$ref": "../string_interpolation/definitions.json#/string_with_placeholders"
+      }
+    },
+    "additionalProperties": false,
+    "required": ["item_title", "title"]
+  },
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/identifier"
+      },
+      "type": {
+        "type": "string",
+        "enum": ["ListCollectorContent"]
+      },
+      "content": {
+        "description": "Content title and contents.",
+        "type": "object",
+        "required": ["title", "contents"],
+        "properties": {
+          "title": {
+            "$ref": "../common_definitions.json#/non_empty_string"
+          },
+          "contents": {
+            "$ref": "../common_definitions.json#/contents"
+          }
+        },
+        "additionalProperties": false
+      },
+      "page_title": {
+        "$ref": "../common_definitions.json#/non_empty_string"
+      },
+      "for_list": {
+        "$ref": "../common_definitions.json#/non_empty_string"
+      },
+      "routing_rules": {
+        "$ref": "../common_definitions.json#/routing_rules"
+      },
+      "repeating_blocks": {
+        "type": "array",
+        "minItems": 1,
+        "items": {
+          "$ref": "list_repeating_block.json#/block"
+        }
+      },
+      "summary": {
+        "$ref": "list_collector_content.json#/summary"
+      }
+    },
+    "additionalProperties": false,
+    "required": ["id", "type", "for_list", "content", "repeating_blocks"]
```

Comment:
Was it agreed that repeating blocks would be mandatory?

Reply:
So here we made an assumption that repeating blocks will be always required? Since otherwise, there will be nothing to display after initial list collector content block?

Reply:
ok cool, makes sense - just checking 🙂

Reply:
We don't need to enforce that this needs repeating blocks. It might have it. It allows them to use it as a summary preview. i.e Here are the list of things you need to report for. Then they might get a repeating section for each